templates/chip/allocator-marshal.html

```diff
@@ -0,0 +1,27 @@
+<section class="{{cssClass}} directory flexcol" id="{{cssId}}" data-tab="{{tabName}}">
+
+  <header class="chip-allocator-header">
+      <div class="flexrow">
+        <h3 class="chip-allocator noborder">{{localize 'DLC.app.ChipAllocator'}}</h3>
+      </div>
+  </header>
+
+  <div class="chip-row">
+    <label class="chip-label-character" for="white">White</label>

```

Comment:
`chip-label-character` defaults to text color `var(--color-text-dark-1)` on my system. The default allocator dialog has a texture background so the black text is visible

Reply:
There are two themes in App v2 apparently, they've added a dark theme. My system is using the light theme, so I didn't see this.

Reply:
right, yes, it's probably set to use the system theme and I have mac switching to dark mode at night.